atomic_reactor/util.py

```diff
@@ -675,17 +679,39 @@ def get_manifest_digests(image, registry, insecure=False, dockercfg_path=None,
         registry = 'https://{}'.format(registry)
 
     context = '/'.join([x for x in [image.namespace, image.repo] if x])
-    tag = image.tag or 'latest'
+    tag = digest or image.tag or 'latest'
```

Comment:
Nit-pick (won't block PR): we could call this `reference` or `ref`, seeing as it could be either a tag or a digest.
cf. https://docs.docker.com/registry/spec/api/#pulling-an-image-manifest